### PR TITLE
fix: Exempted methods does not have error null check for error

### PR DESCRIPTION
**Pull request by Abhishk123**

### CODE REVIEW

File: cypress/support/cypress-commands/assertion.js [hunk 1]
```diff
@@ -175,8 +175,15 @@ Cypress.Commands.add(
...
-            if (!UTILS.getEnvVariable(CONSTANTS.IS_SCENARIO_EXEMPTED, false)) {
+            // Skip the null check for exempted scenarios based on the error response.
+            if (UTILS.getEnvVariable(CONSTANTS.IS_SCENARIO_EXEMPTED, false)) {
+              // Performing error null check validation if the response error is not null.
+              if (response?.error == null) {
+                cy.errorNullCheck(response, CONSTANTS.ERROR).then((result) => {
+                  validationCheck.push(result);
+                });
+              }
+            } else {
```
Comment (neeradanelxsi) [inline cypress/support/cypress-commands/assertion.js:186]: why we need to skip this check if it's an error?
Can't we do similar to negative cases where we are expecting error ? 

[8/18/2024 6:45:05 AM][loggingValidationCheckResult][Error null Check: PASS. Expected error in response: "entityId" is a required property ]

Comment (Abhishk123) [reply cypress/support/cypress-commands/assertion.js:184]: addressed the comment by adding this line

**Review by neeradanelxsi — APPROVED** [review]

**Review by jbigel — APPROVED** [review]

**Pull request merged**